dev-docs/bidder-adaptor.md

```diff
@@ -208,9 +211,9 @@ Sample array entry for `validBidRequests[]`:
 {% endhighlight %}
 
 {: .alert.alert-success :}
-There are several IDs present in the bidRequest object:  
-- **Bid ID** is unique across ad units and bidders.  
-- **Auction ID** is unique per call to `requestBids()`, but is the same across ad units.  
+There are several IDs present in the bidRequest object:
```

Comment:
Hey Matt, unfortunately these spaces are required so that the unordered list that follows is included in the green "box" of the alert.  Can you please drop this part of the changeset?  Otherwise happy to merge ASAP.

Reply:
Oops! My editor trims those whenever I save, configured it not to, let me know if the push I just did works